.github/workflows/regenerate_tidalapi_module.yml

```diff
@@ -0,0 +1,132 @@
+name: Regenerate TidalApi Code
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_branch:
+        description: "The branch to PR against"
+        required: false
+        default: "main"
+env:
+  API_MODULE_DIR: tidalapi
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  openapi-code-generation:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
The other workflows in tidal-sdk-android are already using checkout@v4. It shouldn't be a problem, just fyi for consistency.

Reply:
no you are right. This should be updated. Thanks!